source/core/src/main/com/csse3200/game/areas/terrain/CropTileComponent.java

```diff
@@ -215,7 +216,7 @@ public float getWaterContent() {
 	 *
 	 * @return whether the tile is occupied by the plant
 	 */
-	private boolean isOccupied() {
+	public boolean isOccupied() {
```

Comment:
Was this meant to be changed to public?

Reply:
Yeah, I think we wanted to have it public to begin with for interaction with plant but handled it differently so it worked with it being private. I changed it to public again because of the bug outlined in #419, and the easiest fix was to know if the cropTile had something on it.

Reply:
I can't see it being used outside of `CropTileComponent` anymore in this pr but no worries. 

I've got a different fix in my branch that works with the recent changes Jackson merged into main, I'm happy to just change it back to private there.

Reply:
Sure, that would be great. I honestly didn't bother checking through all the uses of isOccupied that changed while merging (too many conflicts), and also because apparently itemActions and tile interactions are changing again.

Reply:
This was requested for an earlier change to the way tiles interact, but I think team 1 plans on changing tile/terrain interaction again.